php/libraries/Utility.class.inc

```diff
@@ -918,5 +918,31 @@ class Utility
         }
         return $languages;
     }
+
+    /**
+     * Get the list of MRI scan type available in the database
```

Comment:
I am not sure what this function is supposed to do. 
should it return all the scan types from mri_scan_type or all the scan_types where the is at least one file with that scan_type? 
If it's only used in the imaging_browser to limit the number on queries programmatically generated then it should probably be removed. Maintaining this function and the array intersect adds overhead for something that should probably be handle by the frontend anyway.

Reply:
It was first in the provisionner row file but @driusan made a comment about it (https://github.com/aces/Loris/pull/5022#discussion_r313568765) and I moved it the Utility.class.inc. I think he mentioned that this function could also be used by the API? Honestly I don't remember exactly.

@xlecours @driusan could you discuss? I don't have any opinion on what would be best here and will perform the change you suggest after agreement :).

Reply:
Did you link to the right comment? The one you linked to doesn't say anything about the Utility class, it just says the scan types should be passed in via a constructor argument

Reply:
@cmadjar @driusan, I guess it is fine having it in the Utility. I was missing the place where it's also used in the imaging_browser.class.inc